### ISSUE TITLE
DANN Partial Domain Adaption Experiment

### ISSUE THREAD

**Issue by ALshore33** [thuml/Transfer-Learning-Library#31]
Hello Thuml Team,

I tried to run the DANN (ResNet-50) example on the partial domain adaptation setting with OfficeHome/Office31 dataset under the same data protocol as [PADA](https://github.com/thuml/PADA/tree/master/pytorch/data). For Amazon31 to Webcam10 in Office31, the test accuracy starts from 61% and then decreases to 36% after 20 epochs, less than 41.35% reported in the PADA paper and far away from 73.56% in the ETN(CVPR'19) paper.
![dannpda](https://user-images.githubusercontent.com/80360345/110553172-9b496f00-8106-11eb-82a3-cc9273f6c572.png)

I tried close-set Amazon31 to Webcam31, with the same code, and it works just fine with more than 90% accuracy.

Did I do anything wrong with this DANN PDA experiment, or do I need to train the model for more epochs? Thanks.

**Comment by ALshore33** [comment]
Thank you very much for the detailed explanation. I just start learning this wonderful dalib package and the reason I use DANN is it's the simplest model and easy to modify.
